www/docs/customization/release.md

```diff
@@ -14,6 +14,9 @@ release:
   # Repo in which the release will be created.
   # Default is extracted from the origin remote URL or empty if its private hosted.
   # Note: it can only be one: either github, gitlab or gitea
+  # You can use Gitlab's internal project id by setting it directly or using an environment variable
+  # owner: ""
```

Comment:
could you make it more explicit that the owner then needs to be empty in this case